conf/application.conf

```diff
@@ -69,10 +69,14 @@ application.global=uk.gov.hmrc.individualsincomeapi.config.MicroserviceGlobal
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 play.http.router=prod.Routes
 
-# used to encrypt the cache payload
-cache.encryption {
-    key="P5xsJ9Nt+quxGZzB4DeLfw=="
-    previousKeys=[]
+cache {
+    #enabled = false
```

Comment:
I think the default value should be here, so true.

Reply:
👍